harmonic/evidence.pyx

```diff
@@ -404,6 +404,38 @@ class Evidence:
         return (ln_evidence, ln_evidence_std)
 
 
+    def compute_ln_evidence_log_space_var(self):
+        """Compute log_e of evidence from the inverse evidence.
+
+        Returns:
+
+            (double, double): Tuple containing the following.
+
+                - ln_evidence (double): Estimate of log_e of evidence.
+
+                - ln_evidence_std (double): Estimate of standard deviation
+                    of log_e of evidence.
+        
+        """
+
+        self.check_basic_diagnostic()
+
+        ln_x = self.ln_evidence_inv_var - 2.0 * self.ln_evidence_inv
```

Comment:
This line still has ln_x of order ln_evidence_inv, the following line takes the exponent of this which is of order evidence. This may be fine for now, but we should keep in mind that these lines may overflow for large evidences.